examples/jsm/loaders/USDZLoader.js

```diff
@@ -182,8 +182,7 @@ class USDZLoader extends Loader {
 
 					if ( isCrateFile( zip[ filename ] ) ) {
 
-						console.warn( 'THREE.USDZLoader: Crate files (.usdc or binary .usd) are not supported.' );
-						continue;
+						throw Error( 'THREE.USDZLoader: Crate files (.usdc or binary .usd) are not supported.' );
```

Comment:
Yeah, I think throwing an error is indeed more helpful than just logging a warning. In this way, the app can better handle the situation.